.circleci/config.yml

```diff
@@ -45,3 +45,7 @@ workflows:
       - build-and-deploy-image:
           name: rust-build-and-deploy
           imageName: rust
+
+      - build-and-deploy-image:
+          name: therapist-build-and-deploy
+          imageName: therapist
```

Comment:
Instead of calling this `therapist`, what do you think about calling it something like `python-node`? I could see this image being useful even if you're not using Therapist.

Reply:
sure i was thinking in the future this image may add other languages as needed for linting but i'm cool with that

Reply:
Oh, I hadn't thought about having a master therapist image like that. I had originally been thinking about integrating Therapist into every image. Now that you mention it, I think that having a dedicated image for therapist would be a good idea. It also doesn't prevent us from integrating it into the individual images later.

Sorry for flip flopping, but can you put it back?

Reply:
done!